session/session.go

```diff
@@ -0,0 +1,69 @@
+package session
+
+import (
+	"encoding/json"
+	errs "github.com/ONSdigital/dp-sessions-api/apierrors"
+	"github.com/google/uuid"
+	"time"
+)
+
+const (
+	dateTimeFMT = "2006-01-02T15:04:05.000Z"
+)
+
+// Session defines the structure required for a session
+type Session struct {
+	ID    string    `json:"id"`
+	Email string    `json:"email"`
+	Start time.Time `json:"start"`
+}
+
+// CreateSessionEntity is the structure of the request needed to create a session
+type CreateSessionEntity struct {
```

Comment:
I'm beginning to dislike this name (which is my fault not yours). I think something along the lines of `NewSessionDetails` sounds nicer. `SREXDFUYGHIOJOJEntity` feels a bit too Enterprise Java.